phoenix-core/src/it/java/org/apache/phoenix/end2end/ConcurrentMutationsIT.java

```diff
@@ -244,7 +244,7 @@ public void testConcurrentUpserts() throws Exception {
                public void run() {
                    try {
                        Connection conn = DriverManager.getConnection(getUrl());
-                       for (int i = 0; i < 10000; i++) {
+                       for (int i = 0; i < 5000; i++) {
```

Comment:
any particular reason to decrease?

Reply:
The new design is not optimized for excessive number of concurrent writes (and should not be). This test randomly updated the same 51 rows using 4 threads and each of these threads wrote 10000 mutations. These generated large number of unverified index rows and index scan took about 2 minute on my laptop to complete because each of these rows needs to be verified.  The same test is repeated 5 times which made the test takes about 10 minutes. The 10000 number is an arbitrary number and  5000 also creates large number of concurrent mutations. So, I reduced the number to cut the test time by half, to 5 minutes. If this becomes a concern, I can revert it back but the test would take longer.  Before my changes, this test took 2.5 minutes to run.

Reply:
just curious, no need to change, thanks